src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -501,6 +501,15 @@
     <Field Name="LogicalOperator" Type="MethodSymbol"/>
     <Field Name="TrueOperator" Type="MethodSymbol"/>
     <Field Name="FalseOperator" Type="MethodSymbol"/>
+
+    <!-- Used as an operand to bind TrueFalseOperandConversion -->
+    <Field Name="TrueFalseOperandPlaceholder" Type="BoundValuePlaceholder?" SkipInVisitor="true" Null="allow"/>
+    <!--
+    Could be a BoundConversion or, in some cases of an identity conversion, the OperandPlaceholder, or null for no conversion case
+    If the node survives lowering, this property in it must be null.
```

Comment:
Consider asserting that in `LocalRewritingValidator`

Reply:
> Consider asserting that in LocalRewritingValidator

This is an FYI, not a contract or a correctness requirement